html-relative-links/about-me/about-me.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>About Me</title>
+    <style>
+      img {
+        width: 300px;
+      }
+    </style>
+  </head>
+  <body>
+    <ul>
+      <li><a href="../index.html">Home Page</a></li>
+      <li><a href="about-me.html">About Me</a></li>
+      <li><a href="../projects/projects.html">Projects</a></li>
+      <li><a href="../contact-us/contact-us.html">Contact Us</a></li>
+    </ul>
+    <p>Hello About Me!</p>
+    <img src="../images/c-sharp.png" alt="Girl holding programming book" />
```

Comment:
You're using the wrong image here! Compare your example to the example gif

Reply:
thank you Shawn. I have corrected the image links and re-pushed.